js/src/main/scala/mouse/js.scala

```diff
@@ -1,3 +1,24 @@
+/*
+ * Copyright (c) 2022 Typelevel
```

Comment:
@benhutchison the `sbt-typelevel` plugin is bundled with the `sbt-header` plugin, so that's why headers have been generated. The difference with `LICENSE.txt` is that the copyright belongs to `Typelevel`. So we should change that belonging in `LICENSE.txt` or for all files, for consistency. What do you think about that?

Reply:
Happy to ascribe copyright to Typelevel in LICENSE.txt 👍